Bottom: b5305a3f57d85d8a89cb9ca7a1c334939b3a96c3
Top:    ac2f2927bacfbe64bf33761536e429611dd76b2b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 20:05:32 +0000

Refresh of allocate-vnode-with

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 1b59846..217968d 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3950,7 +3950,8 @@ poly_InclVertex (VNODE * after, VNODE * node)
       VNODE *t = PREV_VERTEX (node);
       NEXT_VERTEX (PREV_VERTEX (t)) = node;
       PREV_VERTEX (node) = PREV_VERTEX (t);
-      free (t);
+      g_slice_free (VNODE, t);
+      //free (t);
     }
 }
